File: lintcode/214.max-of-array-vip/max-of-array.java
public class Solution {
    /**
     * @param A: An integer
     * @return: a float number
     */
    public float maxOfArray(float[] A) {
        float max = A[0];
        for (int i = 1; i < A.length; i++) {
            if (A[i] > max) {
                max = A[i];
            }
        }
        return max;
    }
}

File: lintcode/214.max-of-array-vip/max-of-array.go
/**
 * @param A: An integer
 * @return: a float number
 */
 func maxOfArray (A []float32) float32 {
    max := A[0]
    for i:=1;i<len(A);i++{
        if A[i] > max {
            max = A[i]
        }
    }
    return max
}